### PR TITLE
feat(native/networks): update money/networks 2189 natives

### PR DESCRIPTION
**Pull request by ahcenezdh**
* Update/Rename NETWORK_SPEND_UPGRADE_SUB
* Update/Rename NETWORK_SPEND_ISLAND_HEIST
* Update/Rename NETWORK_SPEND_BUY_SUB
* Update/Rename NETWORK_EARN_ISLAND_HEIST
* Update/Rename NETWORK_EARN_FROM_ISLAND_HEIST_DJ_MISSION
* Update/Rename NETWORK_EARN_BEACH_PARTY_LOST_FOUND

Sorry, 4mmonium, I know you said a maximum of 8 file changes, but most of them don't contain documentation, just updates to native names and arguments

### CODE REVIEW

File: MONEY/NetworkEarnFromIslandHeistDjMission.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+```
+NativeDB Introduced: v2189
+```
+
+## Parameters
+* **amount**: The amount of money earned from the DJ mission.
+* **mission**: The identifier of the DJ mission completed.
+
+## Return value
```
Comment (AvarianKnight) [inline MONEY/NetworkEarnFromIslandHeistDjMission.md:22]: If the native doesn't return any value you don't need to include `## Return value`

**Review by AvarianKnight — CHANGES_REQUESTED** [review]

File: MONEY/NetworkEarnFromIslandHeistDjMission.md (hunk 1 above)
Comment (ahcenezdh) [reply MONEY/NetworkEarnFromIslandHeistDjMission.md:22]: Oh didn't know, i'm going to update my PR thanks

File: MONEY/NetworkSpendBuySub.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+ns: MONEY
+aliases: ["0x90CD7C6871FBF1B4", "_NETWORK_SPENT_BUY_SUB"]
```
Comment (4mmonium) [inline MONEY/NetworkSpendBuySub.md:3]: Hi, why is there a `_NETWORK_SPENT_BUY_SUB` alias here? This is not present in the previous native name 🤔

File: MONEY/NetworkEarnBeachPartyLostFound.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+ns: MONEY
+aliases: ["0xA51338E0DCCD4065", "_NETWORK_EARN_FROM_BEACH_PARTY_LOST_FOUND"]
```
Comment (4mmonium) [inline MONEY/NetworkEarnBeachPartyLostFound.md:3]: Same thing here, there is a `_NETWORK_EARN_FROM_BEACH_PARTY_LOST_FOUND` alias that is not present in the previous native documentation anywhere🤔

**Review by 4mmonium — CHANGES_REQUESTED** [review]
Hey there, so far this looks good, requested some slight changes, check my comments. Thanks!

File: MONEY/NetworkSpendBuySub.md (hunk 2 above)
Comment (ahcenezdh) [reply MONEY/NetworkSpendBuySub.md:3]: It's the "old name" of this native found in other website such as nativedb, i'm going to remove it

**Pull request closed without merging**